models/POP/README.rst

```diff
@@ -0,0 +1,440 @@
+|                                     |                   | will be used by the forward observation operators. If the  |
+|                                     |                   | 3rd column is 'UPDATE', the output files will have the     |
+|                                     |                   | modified (assimilated,posterior) values. If the 3rd        |
+|                                     |                   | column is 'NO_COPY_BACK', that variable will not be        |
+|                                     |                   | written to the restart files. **The DART diagnostic files  |
+|                                     |                   | will always have the (modified) posterior values.**        |
+|                                     |                   | Diagnostic variables that are useful for the calculation   |
+|                                     |                   | of the forward observation operator but have no impact on  |
+|                                     |                   | the forecast trajectory of the model could have a value of |
+|                                     |                   | ``NO_COPY_BACK``. The DART kind must be one found in the   |
+|                                     |                   | ``obs_kind_mod.f90`` source code file kept in              |
+|                                     |                   | ``DART/assimilation_code/modules/observations/`` **AFTER** |
+|                                     |                   | it gets built by ``preprocess``. Most of the ocean         |
+|                                     |                   | observation kinds are specified within the                 |
+|                                     |                   | ``obs_def_ocean_mod.f90`` source code file kept in         |
+|                                     |                   | ``DART/observations/forward_operators/``, so it should be  |
+|                                     |                   | specified in the ``&preprocess_nml:input_files``           |
+|                                     |                   | variable.                                                  |
++-------------------------------------+-------------------+------------------------------------------------------------+
+
+.. code-block:: fortran
+
+      namelist /time_manager_nml/  runid, stop_option, stop_count, &
+             time_mix_opt, fit_freq, time_mix_freq, dt_option, dt_count, &
+             impcor, laccel, accel_file, dtuxcel, allow_leapyear, &
+             date_separator, iyear0, imonth0, iday0, ihour0, iminute0, isecond0
+
+This namelist is read in a file called ``pop_in``. This namelist is the same
+one that is used by the ocean model and is used to control the integration
+length of POP.
+
+It is unimportant for the CESM/POP experiments but is critically important for
```

Comment:
Can we highlight the fact this whole section is **NOT NEEDED** for CESM DART/POP?  I question the need to even have it, since it is only for the LANAI version.